### ISSUE TITLE
How to replace src with correct url after upload?

### ISSUE THREAD

**Issue by Dmatafonov** [orthes/medium-editor-insert-plugin#158]
After making upload how do I change image src from base64 to correct url?

**Comment by orthes** [comment]
Duplicate. See #128

**Comment by Dmatafonov** [comment]
worth mentioning in docs
not an obvious solution

**Comment by Dmatafonov** [comment]
you could at least describe "correct" response from server to replace url
looked at UploadHandler.php 

..gush - should I implement All of that just to generate "correct" response?

**Comment by orthes** [comment]
We're adding it to our docs. The plugin uses [blueimp's jQuery File Upload Plugin](https://github.com/blueimp/jQuery-File-Upload) for handling uploads (which is already mentioned in docs). For more info about file upload itself, please refer to their wiki: https://github.com/blueimp/jQuery-File-Upload/wiki

**Comment by Dmatafonov** [comment]
bet someone else will need it

here is how "correct" response can be generated

$files = array();
$files[] = array('url' => "/data/blogs/users/$userId/$newName");
$response = array('files' => $files);
echo json_encode($response);

**Comment by Dmatafonov** [comment]
> The plugin uses blueimp's jQuery File Upload Plugin for handling uploads (which is already mentioned in docs).

I saw it, but I don't get what for you use it. If you rely on css3 and html5, so there is no need for older browsers support and upload cand be done with simple and regular ajax request that works in all browsers and doesn't require any additional libraries. It's up to you. 

Your plugin itself pretty good, but ctrl+v paste needed and simple description on how to handle files serverside. Spent several hours making it work :) And still don't understand how youtube embeds were supposed to work. I had to clean out saved html from '.medium-insert-embeds-overlay' because otherwise I had non-clickable image and iframe beneath..
